payments-core/api/payments-core.api

```diff
@@ -4460,6 +4462,12 @@ public final class com/stripe/android/model/WeChatPayNextAction : com/stripe/and
 	public fun writeToParcel (Landroid/os/Parcel;I)V
 }
 
+public final class com/stripe/android/model/parsers/PaymentIntentJsonParser : com/stripe/android/model/parsers/ModelJsonParser {
```

Comment:
Kind of a bummer that we have to make this public just to use it in a test. I think it's fine, I don't know if we have a better alternative.

Reply:
We can revert this when the newly introduced [java-test-fixtures](http://michaelevans.org/blog/2019/09/21/stop-repeating-yourself-sharing-test-code-across-android-modules/) has better support in Android, so that we can share test code from core to other modules. I tried a couple of hacks to share the code, but non of them are ideal, so ended up this implementation. 
Left a todo in `InjectableActivityScenario`